.github/workflows/pytest.yml

```diff
@@ -0,0 +1,19 @@
+name: Run Python Tests
+
+on: [workflow_dispatch]
```

Comment:
Why workflow triggers manually?

Reply:
Unfortunately test requires a lot of time to run (~3.5h). So, I think we can run them manually when it's really needed. I also added a trigger to run tests when creating a PR to master.

Reply:
Alright, why do we need github actions then?